Gazorator.sln

```diff
@@ -1,39 +0,0 @@
-﻿
```

Comment:
I'd prefer keeping `*.sln` in root. This can be adjusted in `Cake.Recipe` by setting `solutionFilePath` in `BuildParameters`.